1.19.4

matteobruni opened the pull request:
feat: emitters and absorbers now can have a name in options

matteobruni committed:
feat: emitters and absorbers now can have a name in options

ghost commented:
## DeepCode's analysis on [#5188a6](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/31ced50bad71e5ddd7749974ae35357eadfb0dd1/matteobruni/tsparticles/5188a6f6136f923ccbf997991d340a0904348eb0/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=1&) found:

- :information_source: **1** minor issue. :point_down:
## Top issues

<table>
  <thead>
    <tr>
      <th align="left">Description</th>
      <th align="left">Example fixes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td width="77%">'public' is implicit. Occurrences: <ul><li><a href="https://github.com/matteobruni/tsparticles/blob/5188a6f6136f923ccbf997991d340a0904348eb0/core/main/src/Plugins/Emitters/Options/Classes/Emitter.ts#L17">Emitter.ts:17</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/5188a6f6136f923ccbf997991d340a0904348eb0/core/main/src/Plugins/Emitters/Options/Classes/Emitter.ts#L18">Emitter.ts:18</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/5188a6f6136f923ccbf997991d340a0904348eb0/core/main/src/Plugins/Emitters/Options/Classes/Emitter.ts#L19">Emitter.ts:19</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/matteobruni/tsparticles/31ced50bad71e5ddd7749974ae35357eadfb0dd1/matteobruni/tsparticles/5188a6f6136f923ccbf997991d340a0904348eb0/pr/_/%2Fcore%2Fmain%2Fsrc%2FPlugins%2FEmitters%2FOptions%2FClasses%2FEmitter.ts/javascript%2Ftslint%2Fmember-access/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
  </tbody>
</table>


#### 👉 View analysis in [**DeepCode’s Dashboard**](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/31ced50bad71e5ddd7749974ae35357eadfb0dd1/matteobruni/tsparticles/5188a6f6136f923ccbf997991d340a0904348eb0/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=1&) | [_Configure the bot_](https://www.deepcode.ai/app/gh/?ownerconfig=matteobruni)

matteobruni committed:
build: security updates

matteobruni committed:
build: restoring enums to full version, issues with 3rd party packages

matteobruni commented:
This will fix also the issue https://github.com/Wufe/react-particles-js/issues/167

matteobruni committed:
chore(release): published new version

 - ng-particles@2.2.4
 - inferno-particles@1.2.4
 - jquery-particles@1.19.4
 - preact-particles@1.19.4
 - react-tsparticles@1.19.4
 - svelte-particles@1.2.4
 - particles.vue@2.2.4
 - particles.vue3@1.2.4
 - tsparticles-editor@1.19.4
 - tsparticles@1.19.4
 - ng-particles-demo@1.4.4
 - editor-demo@1.4.4
 - inferno-particles-demo@1.2.4
 - jquery-particles-demo@1.4.4
 - tsparticles-demo@1.4.4
 - preact-particles-demo@1.4.4
 - react-tsparticles-demo@1.4.4
 - svelte-demo@1.2.4
 - particles.vue-demo@1.4.4
 - particles.vue3-demo@1.2.4
 - tsparticles-preset-60fps@1.4.4
 - tsparticles-preset-background-mask@1.4.4
 - tsparticles-preset-basic@1.4.4
 - tsparticles-preset-big-circles@1.4.4
 - tsparticles-preset-bouncing@1.4.4
 - tsparticles-preset-fire@1.4.4
 - tsparticles-preset-font-awesome@1.4.4
 - tsparticles-preset-fountain@1.4.4
 - tsparticles-preset-snow@1.4.4
 - tsparticles-preset-stars@1.4.4
 - tsparticles-shape-bubble@1.4.4
 - tsparticles-shape-heart@1.4.4
 - tsparticles-shape-multiline-text@1.4.4
 - tsparticles-shape-spiral@1.4.4
 - cra-template-particles-typescript@1.2.4
 - cra-template-particles@1.2.4
 - tsparticles-website@1.19.4
